version/version_info.go

```diff
@@ -1,49 +1,56 @@
-// Package version provides helpers for working with versions and build info.
 package version
 
+/*
```

Comment:
This code "works", but I would prefer not to use it if we can. It will be apparent if we need it or not soon.

Reply:
Why? Let's remove this file instead of commenting then.

Reply:
After this PR is squashed and merged, this code will be garbage collected by git / GitHub. It will be must harder to recover it.
But after sleeping on it nothing, I'm 90% sure we will not need it. I will keep just in case in the git history but will remove in a next commit.

Reply:
ok